test/jdk/java/util/zip/ZipFile/ReadNonStandardExtraHeadersTest.java

```diff
@@ -562,16 +853,22 @@ public class ReadNonStandardExtraHeadersTest {
     public static void setup() throws IOException {
         Files.deleteIfExists(VALID_APK);
         Files.deleteIfExists(VALID_APACHE_COMPRESS_JAR);
+        Files.deleteIfExists(VALID_ANT_JAR);
+        Files.deleteIfExists(VALID_ANT_ZIP);
+
         // Create the Zip file to read
         Files.write(VALID_APK, VALID_APK_FILE);
         Files.write(VALID_APACHE_COMPRESS_JAR, COMMONS_COMPRESS_JAR);
+        Files.write(VALID_ANT_JAR, ANT_ZIP64_UNICODE_EXTRA_JAR);
+        Files.write(VALID_ANT_JAR, ANT_ZIP64_UNICODE_EXTRA_ZIP);
+
     }
 
     /**
      * Zip and Jars files to validate we can open
      */
     private static Stream<Path> zipFilesToTest() {
-        return Stream.of(VALID_APK, VALID_APACHE_COMPRESS_JAR);
+        return Stream.of(VALID_APK, VALID_APACHE_COMPRESS_JAR, VALID_ANT_JAR);
```

Comment:
And here you probably want to add `VALID_ANT_ZIP` in addition to `VALID_ANT_JAR`.

Reply:
Yep, already caught that typo, forgot to save before I committed :-)